codes.md

```diff
@@ -0,0 +1,30 @@
+Formatting conventions
+======================
+
+## Codes and categories should be written as first-level headers
+  - Discrete supporting evidence
+  - (or codes associated with categories)
+  - are listed as distinct bullet points
+
+Codes
+=====
+## Participants often want to add additional information, data that isn’t there
+  - Impose structure/hierarchy: for the junk drawer, some participants group items (Session 013, Session 007). For the file system, a participant wanted to add names that reflected a folder hierarchy (Session 012)
+  - Add data to each item: for the file system, a participant wanted to add the file sizes (Session 009). For the power plants, participants want to reduce down to the number of people or to the number of individual units (Session 014)
+  - Observation by KI: Want to add a 'richness' to the data in lots of dimensions. This isn't their data so they want to put it into the context of what they know.
+
+## Participants want to uncover the purpose: “Why is the file system organized like this?,” “what are the power stations doing?”
+  - Make a story/context for the file system: Session 006, Session 009, Session 012
+
+## Participants want to discover ultima: For power plant, which one produces the most energy?   
+  - Supported by Session 011, Session 014, not Session 008
+
+## The idea that data relates to computers is prevalent: 
+  - Session 010 wrote a Python dictionary. 
+  - Sessions 006, 009, 012, 015 used file system icons that they cited as being from an operating system.
+  - Reliance on tables: Session 007 (“I guess .. table”) and Session 014
+
+## Physical objects can also be used to represent data
```

Comment:
Really curious to see how this code evolves; in a sense, with the junk drawer, we're kind of doing *reverse* data physicalization.

Probably no way to tell since we're asking people to *draw* in the first place, but maybe there's something here about concrete icons / emoji (ooh, by extension: word-scale vis?) sometimes being a more natural way for people to describe parts of a data abstraction, than it is to use words or abstract diagrams?

i.e. if I expect an upward trend to exist over time, 📈  might be an easier way to say it than "I expect an upward trend to exist over time"

Reply:
Good observation. I've been surprised with how many drawings of objects that we've gotten. Maybe it's more fun? But it's not how I would communicate those items and their quantities (although that would also depend on my goal: am I showing how I'd organize these or telling someone what to get at Office Max?)